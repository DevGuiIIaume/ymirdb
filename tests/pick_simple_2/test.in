set a 1 2 3
pick a 2
del a
pick a 2
bye
