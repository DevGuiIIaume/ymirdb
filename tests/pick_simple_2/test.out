> ok

> 2

> ok

> no such key

> bye
